shared/surface_flux.F90

```diff
@@ -173,12 +176,17 @@ module surface_flux_mod
                                            !! heat.  This option is available for legacy purposes, and is not recommended for
                                            !! new experiments.
 logical :: ncar_ocean_flux_multilevel  = .false. !< Use NCAR climate model turbulent flux calculation described by Large and Yeager, allows for different reference height for wind, temp and spec. hum.
+logical :: do_iter_monin_obukhov       = .false. !< If .TRUE,  call monin obukhov funtions a couple of times to update 
+                                                 !! rough_mom, rough_heat, rough_moist, cd, ch, b_star, u_star 
+logical :: use_u10_neutral             = .false. !< If .TRUE., use 10m neutral wind rather than the standard 10m wind 
+                                                 !! to obtain rough_mom, rough_heat, rough_moist
 real :: bulk_zu = 10.                      !< Reference height for wind speed (meters)
 real :: bulk_zt = 10.                      !< Reference height for atm temperature (meters)
 real :: bulk_zq = 10.                      !< Reference height for atm humidity (meters)
 logical :: raoult_sat_vap        = .false. !< Reduce saturation vapor pressure to account for seawater
 logical :: do_simple             = .false.
-
+integer :: niter                 = 5       !< iteration times to call iter_monin_obukhov_ocean. Typically 3-5 times should converge
```

Comment:
Would like to see this be more descriptive to match the trigger `do_iter_monin_obukhov`, perhaps `niter_monin_obukhov`.  If updated, please change all instances.